lib/byte_safe_strings.php

```diff
@@ -50,8 +50,10 @@ function RandomCompat_strlen($binary_string)
                     'RandomCompat_strlen() expects a string'
                 );
             }
+
             return mb_strlen($binary_string, '8bit');
         }
+
```

Comment:
I know that, but it's not the point.

This very PR is obviously intended to make such changes, which shouldn't happen. Everybody should just stop assuming that *their* style is what everybody wants.